controllers/accounts.go

```diff
@@ -13,18 +13,22 @@ import (
 	opaqueMsg "github.com/bytemare/opaque/message"
 	"github.com/go-chi/chi/v5"
 	"github.com/go-chi/render"
+	"github.com/google/uuid"
 )
 
 type AccountsController struct {
 	opaqueService *services.OpaqueService
 	jwtService    *services.JWTService
+	twoFAService  *services.TwoFAService
 	ds            *datastore.Datastore
 }
 
 // @Description Response for password setup or change
 type PasswordFinalizeResponse struct {
 	// Authentication token
 	AuthToken string `json:"authToken"`
+	// Indicates if 2FA verification is required before password setup is complete
```

Comment:
Just making it clear that this is a client hint (for improved UX).
```suggestion
	// Indicates to the client whether 2FA verification will be required before password setup is complete
```